ai/STA/Tactic/align_to_defense_wall.py

```diff
@@ -149,4 +157,15 @@ def _is_closest_not_goaler(self, player):
         return closest_players[0].player == self.game_state.get_player_by_role(Role.GOALKEEPER) \
                and player == closest_players[1].player
 
+    def _closest_point_away_from_ball(self):
+        inters = intersection_line_and_circle(self.game_state.ball_position, KEEPOUT_DISTANCE_FROM_BALL,
```

Comment:
It's not handle since the only time it's call, is when `self.center_formation` is inside the circle. If `center_formation` is inside the circle, there will be two intersections.